v1/bitbucket/project/long-live-tokens.rego

```diff
@@ -0,0 +1,67 @@
+package verify
+
+import future.keywords.in
+
+default allow := false
+
+default violations := []
+
+default max_days := 30
+
+verify = v {
+    v := {
+        "allow": allow,
+        "violation": {
+            "type": "long live tokens",
+            "details": violations,
+        },
+        "summary": [{
+            "allow": allow,
+            "reason": reason,
+            "violations": count(violations),
+        }],
+    }
+}
+
+allow {
+    count(violations) == 0
+}
+
+max_days := input.config.args.max_days {
+    input.config.args.max_days
+}
+
+nanosecs_per_second := (1000 * 1000) * 1000
+nanosecs_per_day := ((24 * 60) * 60) * nanosecs_per_second
+max_lifetime := max_days * nanosecs_per_day
+
+adjust_timezone(ts) := adj_ts {
+    ts_with_t := replace(ts, " ", "T")
+    adj_ts := ts_with_t
+    contains(ts, "+")
+}
+
+adjust_timezone(ts) := adj_ts {
+    ts_with_t := replace(ts, " ", "T")
+    not contains(ts, "+")
+    adj_ts := concat("", [ts_with_t, "Z"])
+}
```

Comment:
Are you sure it's not implemented in `time.format()`?
https://www.openpolicyagent.org/docs/latest/policy-reference/#time